ocaml/utils/language_extension.ml

```diff
@@ -65,6 +65,34 @@ let get_level_ops : type a. a t -> (module Extension_level with type t = a) =
   | Labeled_tuples -> (module Unit)
   | Small_numbers -> (module Unit)
 
+module Exist_pair = struct
+  include Exist_pair
+
+  let maturity : t -> Maturity.t = function
+    | Pair (Comprehensions, ()) -> Beta
+    | Pair (Mode, ()) -> Stable
+    | Pair (Unique, ()) -> Alpha
+    | Pair (Include_functor, ()) -> Stable
+    | Pair (Polymorphic_parameters, ()) -> Stable
+    | Pair (Immutable_arrays, ()) -> Stable
+    | Pair (Module_strengthening, ()) -> Stable
+    | Pair (Layouts, m) -> m
+    | Pair (SIMD, ()) -> Stable
+    | Pair (Labeled_tuples, ()) -> Stable
+    | Pair (Small_numbers, ()) -> Alpha
```

Comment:
Yeah, that's right, this is is meant to replace `default_extensions`. Bumping `SIMD` and `Module_strengthening` is also intentional and was discussed.

I'm not sure we should update `default_extensions`, but I'd prefer to delay it so `SIMD` and `Module_strengthening` are enabled on the update to the build rules, not to the compiler. Given that, if it's not kept in sync, I also don't think we should create a separate PR for that, since we're not changing the default but rather creating a new setting.

Perhaps we should rename `default_extensions` to something like `legacy_default_extensions`?

Reply:
I’m a little hazy on the details here, but if we’re going to have a `legacy` something, I’d love to have a plan to get rid of it. Even if the plan gets put off roughly indefinitely because it’s low priority.

Reply:
The plan is to delete it together with `-extension` after we fix every mention of `-external` in build rules :)

Reply:
I'm happy to sacrifice this part of your plan to accomplish that:

> I'd prefer to delay it so SIMD and Module_strengthening are enabled on the update to the build rules, not to the compiler.

Reply:
I think of it as:

1) Not passing `-extension-universe` (and maybe passing `-extension ...`) is the "legacy" mode used by current build rules. I want to keep it the same until we switch everyone to the new mode.
2) Passing `-extensions-universe` is the new mode, in which you set what universe you're in the jbuild/dune field. `-extension` is a de-facto no-op until we remove it.

It seems fine for the new mode to be semantically different from the old mode?

Reply:
Enabled SIMD and module strengthening by default.

Reply:
For context here, Diana and I discussed. My stance was that we should making "passing no arguments" and "passing `-universe Stable`" line up, and I asked her to make that change.